Sources/XcodesKit/RuntimeList.swift

```diff
@@ -47,6 +51,93 @@ public class RuntimeList {
             return first.identifier.uuidString.compare(second.identifier.uuidString, options: .numeric) == .orderedAscending
         }
     }
+
+    public func downloadAndInstallRuntime(identifier: String, downloader: Downloader) async throws {
+        let downloadables = try await downloadableRuntimes(includeBetas: true)
+        guard let matchedRuntime = downloadables.first(where: { $0.visibleIdentifier == identifier }) else {
```

Comment:
So one concern I have is the onus on the user to have to put quotes around the version input. 

Wondering how we could make that easier? Or allow alternative options? 
- Allow `runtimes install ios 16.0` where it parses both. 
- Have no space `iOS16.0`?

Reply:
I think requiring users to quote input with spaces is reasonable, especially since that's a pattern already established by many CLI tools (e.g. changing to a directory that includes a space in the name). I think the two proposed alternatives aren't desirable.

- A single input (in this case version) should be consumed as a single argument, not _n_ number of arguments. This would also prevent future extensibility of the command, such as specifying multiple versions to install in one command.
- Listing available runtimes shows a list where versions have spaces in the name (e.g. "ios 16.0"). Requiring users to input a version without spaces (e.g. "ios16.0") makes it harder to view available runtimes and then install them. It would also require documenting this need well enough for users to understand, in which case you might as well just say 'put quotes around the version'.

Reply:
@MattKiazyk I tend to agree with @beechtom on this. The first option IMO is doing more harm than good, and the second option is partially applied in beta versions `XX.betaX` but not the platform name itself because I don't think it looks good. I hope that doesn't block the PR from being merged.

Reply:
@StevenMagdy no it won't block it. I'm on the minority when it comes using a CLI, so if others think that's the normal, i'm good